scripts/prepare-release.sh

```diff
@@ -2,30 +2,50 @@
 set -e
 set -o pipefail
 
-PREVIOUS_VERSION=$(grep integreatly-operator deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | awk -F v '{print $2}')
+if [[ -z "$TYPE" ]]; then
+  TYPE="integreatly-operator"
+fi
+
+PREVIOUS_VERSION=$(grep $TYPE deploy/olm-catalog/$TYPE/$TYPE.package.yaml | awk -F v '{print $2}') || echo "No previous version"
```

Comment:
could we use jq here to pick out the currentCSV version? if there were any other instances of the $TYPE entry in the file this would pick up on it as well.

Something like

`yq r <file> channels[0].currentCSV`

I'm not super familiar with the manifests in integreatly-operator but I'm pretty sure we don't ever include historical channels so could fairly safely assume it's the first entry.

Reply:
This is just an update to the initial script to allow for parameters and didn't see the need to rewrite the whole thing.

On a personal I don't like the use of `yq` as its not a stand tool on most systems. But that is my personal opinion and would be will to change.